dask/dataframe/tests/test_rolling.py

```diff
@@ -383,7 +385,14 @@ def test_rolling_agg_aggregate():
 
 @pytest.mark.skipif(not dd._compat.PANDAS_GT_100, reason="needs pandas>=1.0.0")
 def test_rolling_numba_engine():
-    pytest.importorskip("numba")
+    numba = pytest.importorskip("numba")
+    if (
+        dd._compat.PANDAS_VERSION <= "1.0.3"
```

Comment:
Can you define `dd._compat.PANDAS_GT_103` use use `dd._compat.PANDAS_GT_103` here? Makes it easier to clean up later since we just need to find that symbol.

Reply:
Sounds good. I used `PANDAS_GT_104` to be consistent with the other `PANDAS_GT_*` variables which use `>=` for version restrictions.